pcsx2/Frontend/InputManager.cpp

```diff
@@ -1190,14 +1209,16 @@ void InputManager::ReloadBindings(SettingsInterface& si, SettingsInterface& bind
 	for (u32 pad = 0; pad < PAD::NUM_CONTROLLER_PORTS; pad++)
 		AddPadBindings(binding_si, pad, PAD::GetDefaultPadType(pad));
 
-	for (u32 axis = 0; axis < static_cast<u32>(InputPointerAxis::Count); axis++)
+	const float ui_ctrl_range = 100.0f;
+	const float pointer_sensitivity = 0.05;
+	for (u32 axis = 0; axis <= static_cast<u32>(InputPointerAxis::Y); axis++)
 	{
-		// From lilypad: 1 mouse pixel = 1/8th way down.
-		const float default_scale = (axis <= static_cast<u32>(InputPointerAxis::Y)) ? 8.0f : 1.0f;
-		s_pointer_axis_scale[axis] =
-			1.0f /
-			std::max(si.GetFloatValue("Pad", fmt::format("Pointer{}Scale", s_pointer_axis_names[axis]).c_str(), default_scale), 1.0f);
+		s_pointer_axis_speed[axis]     = si.GetFloatValue("Pad", fmt::format("Pointer{}Speed", s_pointer_axis_names[axis]).c_str()) / ui_ctrl_range * pointer_sensitivity;
+		s_pointer_axis_dead_zone[axis] = std::min(si.GetFloatValue("Pad", fmt::format("Pointer{}DeadZone", s_pointer_axis_names[axis]).c_str()) / ui_ctrl_range, 1.0f);
+		s_pointer_axis_range[axis]     = 1.0f - s_pointer_axis_dead_zone[axis];
 	}
+	s_pointer_inertia = si.GetFloatValue("Pad", fmt::format("PointerInertia").c_str()) / ui_ctrl_range;
```

Comment:
This is missing a default value (and might be responsible for what Jordan said).

Reply:
I'll check that, thanks

Reply:
what I am actually seeing right now is that the emulator won't load any custom profile, but only the "shared" profile will work, even if I create the new profile from scratch. Can someone lend some help?

Reply:
Did you change it in 2 locations? Globally:
![image](https://user-images.githubusercontent.com/24227051/217098792-6a54acda-e7f5-4230-94f8-65e6266815f2.png)

Then per-game:
![image](https://user-images.githubusercontent.com/24227051/217098885-e16f30a2-b37e-445b-8c08-511f729bd8d2.png)

Unless that doesn't work with the USB devices.

Reply:
> Unless that doesn't work with the USB devices.

#8051

Reply:
Oh, nvm thought that was merged already.

Reply:
> I'm not sure I follow, you mean a default value where it is declared as float at the top of the file?

No, the third parameter to `GetFloatValue()` is what is returned when the key does not exist in the config. (also, the fmt is redundant there)

Reply:
Ok thanks. Sorry for the extremely long reaction times, life has a way of getting in the way of what's truly important :)